spec/dish_printer_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'dish_printer'
+
+describe DishPrinter do
+  describe '#print' do
+
+    it 'prints the name and price of the dish' do
+      dish_double = double('dish', name: 'Yummy', price: 9.99)
+      expect(subject.print(dish_double)).to eq('Yummy 9.99')
```

Comment:
I believe you can use the name of the module as subject is usually used to reference a class instance.

e.g. expect(DishPrinter.print(dish_double)).to eq('Yummy 9.99')